bigtable/reader.go

```diff
@@ -138,13 +140,15 @@ func (cr *chunkReader) handleCellValue(cc *btpb.ReadRowsResponse_CellChunk) Row
 		// ValueSize is specified so expect a split value of ValueSize bytes
 		if cr.curVal == nil {
 			cr.curVal = make([]byte, 0, cc.ValueSize)
+			cr.curLabels = cc.Labels
 		}
 		cr.curVal = append(cr.curVal, cc.Value...)
 		cr.state = cellInProgress
 	} else {
 		// This cell is either the complete value or the last chunk of a split
 		if cr.curVal == nil {
 			cr.curVal = cc.Value
+			cr.curLabels = cc.Labels
```

Comment:
According to the docs labels are sent in the first cell chunk. Thus, we're taking them when the first chunk of the cell is read